db/migrate/20160820205817_create_publishers.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePublishers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :publications, id: :uuid  do |t|
+      t.string :name
+      t.string :type
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
possibly a git thing.  let me try it again and see what happens.

On Tue, Aug 23, 2016 at 9:53 AM, Brian Kung <notifications@github.com>
wrote:

> In db/migrate/20160820205817_create_publishers.rb
> <https://github.com/briankung/projectcredo/pull/64#discussion_r75882861>:
>
> > @@ -0,0 +1,10 @@
> > +class CreatePublishers < ActiveRecord::Migration[5.0]
> > +  def change
> > +    create_table :publications, id: :uuid  do |t|
> > +      t.string :name
> > +      t.string :type
> > +
> > +      t.timestamps
> > +    end
> > +  end
> > +end
>
> That doesn't seem right. Are you sure it wasn't a git thing? I'm 99% sure
> Rails doesn't do that.
> … <#m_7746067793841332578_>
> On Tue, Aug 23, 2016 at 10:51 AM, wsf246 ***@***.***> wrote: In
> db/migrate/20160820205817_create_publishers.rb <#64 (comment)
> <https://github.com/briankung/projectcredo/pull/64#discussion_r75882389>>:
> > @@ -0,0 +1,10 @@ > +class CreatePublishers < ActiveRecord::Migration[5.0]
> > + def change > + create_table :publications, id: :uuid do |t| > +
> t.string :name > + t.string :type > + > + t.timestamps > + end > + end >
> +end I tried that but the migration file name reverted back on the migrate,
> unless I did something funky. I think it's some rails magic that ensures
> you don't fuck with migrations. — You are receiving this because you
> commented. Reply to this email directly, view it on GitHub <
> https://github.com/briankung/projectcredo/pull/64/files/
> 2b8cccb48a7d372bd7c56a40f6732f85f108ff4b#r75882389>, or mute the thread <
> https://github.com/notifications/unsubscribe-auth/ACtGx1WpixBULXC_
> mN6FkRfi1CbElyV0ks5qiwjegaJpZM4JqR56> .
> -- Brian Kung http://about.me/briankung/ <http://www.facebook.com/
> brian.kung> <http://www.linkedin.com/in/briankung> <
> http://www.twitter.com/briankung> BE A UNICORN!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/64/files/2b8cccb48a7d372bd7c56a40f6732f85f108ff4b#r75882861>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACKsZHxOwd4tPQon081LzQrdRXFt8yKzks5qiwlZgaJpZM4JqR56>
> .
>